google/cloud/dataproc_spark_connect/session.py

```diff
@@ -354,6 +354,11 @@ def _get_dataproc_config(self):
                 dataproc_config = self._dataproc_config
                 for k, v in self._options.items():
                     dataproc_config.runtime_config.properties[k] = v
+            # Ensure the legacy setCommandRejectsSparkCoreConfs is always false
+            # for the Dataproc session configuration, consistent with the setting
+            # in dataprocSessionConfig. This disables an incorrect warning check.
+            # See: https://spark.apache.org/docs/latest/sql-migration-guide.html#ddl-statements
+            dataproc_config.runtime_config.properties["spark.sql.legacy.setCommandRejectsSparkCoreConfs"] = "false"
```

Comment:
I think that Dataproc properties are set when we create a Dataproc Session that Spark Connect client should not be aware of. If this is the case, then how Spark client picks them up to show a warning? Ideally we would fix root cause, instead of suppressing warning message, which this property seems to do.